src/pyodide/internal/metadatafs.ts

```diff
@@ -2,9 +2,9 @@
 import { default as MetadataReader } from "pyodide-internal:runtime-generated/metadata";
 import { createReadonlyFS } from "pyodide-internal:readOnlyFS";
 
-function createTree(paths) {
+function createTree(paths: string[]): Map<string, object> {
```

Comment:
The `object` here is really a recursive type that's effectively another Map, right? Can it be defined like so?